src/Badges.sol

```diff
@@ -270,6 +273,28 @@ contract Badges is
     return mint(msg.sender, _uri, raftTokenId);
   }
 
+  function airdropWithConsent(
+    address _recipient,
+    string calldata _uri,
+    bytes calldata _issuerSignature,
```

Comment:
Why not just use AgreementHash for issuer signature and RequestHash for claimant signature? Wouldn't that give us what we need? i.e., 

Issuer essential is saying "I approve this badge for this claimant"
Claimant is essentially saying "I consent to receiving this badge"

With those 2 signatures, anyone can airdrop it, no?

Reply:
yep agreed. Updated